### PR TITLE
Fixed the bug that I had fixed before

### PR DESCRIPTION
**Pull request by Vane11ope**
GIFrecord_2019-08-11_195554
GIFrecord_2019-08-11_195344

before and after

### CODE REVIEW

File: libr/core/panels.c [hunk 1]
```diff
@@ -1862,14 +1865,22 @@ bool __handle_mouse(RCore *core, RPanel *panel, int *key) {
...
-				__handle_mouse_on_menu (core, x, y);
+			if (panels->mode == PANEL_MODE_MENU && __trigger_mouse_on_menu (core, x, y)) {
+				panels->mouse_cb = __handle_mouse_on_menu;
+				panels->mouse_cb_x= x;
+				panels->mouse_cb_y= y;
 				return true;
 			}
-			if (__handle_mouse_on_X (core, x, y)) {
+			if (__trigger_X (core, x, y)) {
+				panels->mouse_cb = __handle_mouse_on_X;
```
Comment (radare) [inline libr/core/panels.c]: Wrong spacings around =

File: libr/core/panels.c [hunk 2]
```diff
@@ -1920,33 +1936,57 @@ bool __handle_mouse_on_top (RCore *core, int x, int y) {
...
+bool __trigger_X(RCore *core, int x, int y) {
 	RPanels *panels = core->panels;
 	const int idx = __get_panel_idx_in_pos (core, x, y);
-	char *word = get_word_from_canvas (core, panels, x, y);
 	if (idx == -1) {
 		return false;
 	}
 	RPanel *ppos = __get_panel(panels, idx);
 	const int TITLE_Y = ppos->view->pos.y + 2;
+	if (y == TITLE_Y) {
```
Comment (radare) [inline libr/core/panels.c]: Just return y == TITLE_Y

**Comment by Vane11ope** [comment]
ok hang on

**Comment by Vane11ope** [comment]
DONE

File: libr/core/panels.c [hunk 3]
```diff
@@ -450,10 +450,12 @@ static void __toggle_window_mode(RCore *core);
 
 /* mouse */
 static bool __handle_mouse(RCore *core, RPanel *panel, int *key);
-static bool __handle_mouse_on_top(RCore *core, int x, int y);
-static void __handle_mouse_on_menu(RCore *core, int x, int y);
-static bool __handle_mouse_on_X(RCore *core, int x, int y);
+static void __handle_mouse_on_top(void *user, int x, int y);
```
Comment (radare) [inline libr/core/panels.c]: Why void, rcore is more clear

File: libr/core/panels.c [hunk 4]
```diff
@@ -792,6 +794,7 @@ char *__show_status_input(RCore *core, const char *msg) {
 	char *n_msg = r_str_newf (R_CONS_CLEAR_LINE"%s[Status] %s"Color_RESET, core->cons->context->pal.graph_box2, msg);
 	r_cons_gotoxy (0, 0);
 	r_cons_flush ();
+	fflush (stdout);
```
Comment (radare) [inline libr/core/panels.c]: Shouldnt be necessary

File: libr/core/panels.c (hunk 3 above)
Comment (Vane11ope) [reply libr/core/panels.c]: need you to read the code and there is a reason

File: libr/core/panels.c (hunk 4 above)
Comment (Vane11ope) [reply libr/core/panels.c]: good catch

File: libr/core/panels.c (hunk 3 above)
Comment (radare) [reply libr/core/panels.c]: Thats what i did and still i dont see any reason for that change when u are blindcasting to rcore this void* in the first line of each of those functions

Comment (radare) [reply libr/core/panels.c]: ping? ill merge when i understand this change because i  dont s ee a reason  to remove core here

**Comment by radare** [comment]
tired of waiting for an answer. merging ill just fix master later if needed, but this void+ is wrong

**Pull request merged**